docs/source/models/other_models/pregnancy/gbd_2021_mncnh/index.rst

```diff
@@ -0,0 +1,335 @@
+
+.. list-table:: State prevalence table for initialization
+  :widths: 15 15 15
+  :header-rows: 1
+
+  * - State
+    - Value
+    - Note
+  * - np
+    - 0
+    - 
+  * - p
+    - 1
+    - 
+
+.. list-table:: State transition data
+  :header-rows: 1
+
+  * - Source state
+    - Sink state  
+    - Transition name
+    - Transition rate
+    - Note
+  * - np
+    - p
+    - incidence_p
+    - 0
+    - Assumed zero for convenience. Note that this is typically :math:`\frac{ASFR + ASFR * SBR + incidence_\text{c995} + incidence_\text{c374}}{prevalence_\text{np}}`
+  * - p
+    - output
+    - completion of the decision tree
+    - Time step based transition where all simulants move to the next stage following the decision tree
+    - Single time step based state 
+
+.. list-table:: Data values
+  :header-rows: 1
+
+  * - Parameter
+    - Data type  
+    - Data ID
+    - Source
+    - Note
+  * - ASFR
+    - Covariate
+    - 13
+    - get_covariate_estimates: decomp_step='iterative' for GBD 2021
+    - Assume lognormal distribution of uncertainty.
+  * - SBR
+    - Covariate
+    - 2267
+    - get_covariate_estimates: decomp_step='iterative' for GBD 2021
+    - Parameter is not age specific and has no draw-level uncertainty. Use mean_value as location-specific point parameter.
+  * - incidence_c995
+    - Incidence rate of abortion and miscarriage cause
+    - c995
+    - como; decomp_step='iterative'
+    -  Use the :ref:`total population incidence rate <total population incidence rate>` directly from GBD and do not rescale this parameter to susceptible-population incidence rate using condition prevalence. 
+  * - incidence_c374
+    - Incidence rate of ectopic pregnancy
+    - c374
+    - como; decomp_step='iterative'
+    -  Use the :ref:`total population incidence rate <total population incidence rate>` directly from GBD and do not rescale this parameter to susceptible-population incidence rate using condition prevalence. 
+
+.. list-table:: Restrictions
+   :widths: 15 15 20
+   :header-rows: 1
+
+   * - Restriction Type
+     - Value
+     - Notes
+   * - Male only
+     - False
+     -
+   * - Female only
+     - True
+     -
+   * - Age group start
+     - 10 to 14 years
+     - ID=7
+   * - Age group end
+     - 50 to 54 years
+     - ID=15
+
+
+**Pregnancy modeling steps:**
+
+*At initialization:*
+
+#. Assign pregnancy state according to state prevalence values
+#. Assign partial or full term duration according to table in `Pregnancy term lengths`_ section
+#. Assign sex of infant if pregnancy is full term (stillbirth or live birth)
+#. Assign duration of pregnancy depending on term length and, if applicable, sex of the infant. Note that this is the same value as "gestational age" in other parts of the documentation.
+#. Assign birthweight of simulant child 
+#. Assign propensity values for ANC and ultrasound 
+#. Begin simulation
+
+
+See below for more detailed information on how to assign several of these items. 
```

Comment:
I think it would be clearer if some version of this comment were moved to in front of the list of steps. I started reading the list and was like, "Wait, how are you supposed to do that?" before realizing that all the details are below instead of above. If we want to summarize all the steps before giving all the details, I think we should start the section with a comment like, "Here's a summary of the algorithm for the pregnancy model. Details for each step are provided below," or something like that.